buildlib/azure-pipelines.yml

```diff
@@ -136,6 +136,25 @@ stages:
               ./contrib/buildrpm.sh -s -t -b
             displayName: Build tarball
 
+      # Publish JUCX to maven cental
+      - job: publish_jucx
+        displayName: Publish JUCX SNAPSHOT artifact to maven central
+        container: fedora
+        #condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
Overall PR looks good to me, let's uncomment this one.

Reply:
👍 Want to make sure it'll publish from fedora containers.

Reply:
Then will uncomment back.